### ISSUE TITLE
Gene IDs in `fbc:listOfGeneProducts` and `fbc:geneProductAssociation` don't match in Fruitfly-GEM.xml

### ISSUE THREAD

**Issue by htpusa** [SysBioChalmers/Fruitfly-GEM#10]
### Description of the issue:
The gene IDs used to define GPR in `fbc:geneProductAssociation` don't have the "G_" prefix, while the genes listed in `fbc:listOfGeneProducts` do. This mismatch causes problems when trying to use GPRs, for example, in COBRA toolbox.

#### Expected feature/value/output:
An example:

```
<fbc:geneProductAssociation>
          <fbc:geneProductRef fbc:geneProduct="CG2767"/>
</fbc:geneProductAssociation>
```

and

```
 <fbc:geneProduct metaid="CG2767" sboTerm="SBO:0000243" fbc:id="CG2767" fbc:label="CG2767"/>
```

#### Current feature/value/output:
```
<fbc:geneProductAssociation>
          <fbc:geneProductRef fbc:geneProduct="CG2767"/>
</fbc:geneProductAssociation>
```

and

```
 <fbc:geneProduct metaid="G_CG2767" sboTerm="SBO:0000243" fbc:id="G_CG2767" fbc:label="G_CG2767"/>
```

#### Reproducing these results:
Example of the problems encountered using COBRA toolbox:

```matlab
>> model=readSBML('model/Fruitfly-GEM.xml',10);
The model contains 0 errors and 1 warnings.

>> % GPR for first reaction refers to genes number 1811 and 1812
model.rules(1)

ans =

  1×1 cell array

    {'( x(1811) | x(1812) )'}

>> % but there are only 1810  genes in the model
numel(model.genes)

ans =

        1810

>> % when trying to create the grRules field
creategrRulesField(model)
Error using regexprep
Evaluation of 'model.genes{str2num($1)}' failed:

Index exceeds the number of array elements. Index must not exceed 1810.

Error in creategrRulesField (line 46)
currentrules = regexprep(currentrules,'x\(([0-9]+)\)','${model.genes{str2num($1)}}');

```

**I hereby confirm that I have:**
- [ x] Done this analysis in the `main` branch of the repository
- [ x] Checked that a similar issue does not exist already

**Comment by haowang-bioinfo** [comment]
@htpusa thanks for reporting this mismatch, will see possible fix

**Comment by haowang-bioinfo** [comment]
besides, a quick alternative could be just directly loading the Matlab file by command:
```matlab
load('model/Fruitfly-GEM.mat');

% and then convert to Cobra format:
model = ravenCobraWrapper(fruitflyGEM);
```

**Comment by htpusa** [comment]
Yes, this seems to work.

**Comment by haowang-bioinfo** [comment]
fixed in [v1.3](https://github.com/SysBioChalmers/Fruitfly-GEM/releases/tag/v1.3.0)
